src/ncp/posix/netif.hpp

```diff
@@ -46,6 +46,12 @@
 
 namespace otbr {
 
+#if defined(RTM_NEWMADDR) || defined(__NetBSD__)
+#define NETIF_USE_MLD_MONITOR 0
```

Comment:
This is not following the naming style of OTBR

Reply:
Updated, thanks!